src/libbitcoind.cc

```diff
@@ -1566,7 +1566,10 @@ NAN_METHOD(GetMempoolOutputs) {
 
             Local<Object> output = NanNew<Object>();
 
-            output->Set(NanNew<String>("script"), NanNew<String>(script.ToString()));
+            output->Set(NanNew<String>("address"), NanNew<String>(psz));
```

Comment:
Considering that we could pass in multiple addresses, this may make sense later.